core/src/Revolution/Rest/modRestClient.php

```diff
@@ -161,12 +161,12 @@ public function setResponseType($type) {
      * Translates a SimpleXMLElement object into an array.
      *
      * @access public
-     * @param SimpleXMLElement $obj
+     * @param \SimpleXMLElement $obj
      * @param array &$arr The reference array to store the results in.
      * @return boolean True if successful.
      */
     public function xml2array($obj, &$arr) {
-        if (!($obj instanceof SimpleXMLElement)) return false;
+        if (!($obj instanceof \SimpleXMLElement)) return false;
```

Comment:
I'd like to go with imports instead of referencing classes from the global namespace this way.

Reply:
I will change them in a few hours 👍